test/test_centos_common_support.py

```diff
@@ -7,23 +7,6 @@
 )
 
 
-def test_release_supported_version_check_centos(host):
```

Comment:
as @yubiuser mentioned, we could update tests for `unsupported_dialog`.

Reply:
Actually, I was thinking about this.. why not just remove the unsupported dialog all together? We don't have a special case for various flavours of debian/ubuntu - just `PIHOLE_SKIP_OS_CHECK` and hope for the best... why hold the hands of  RHEL-like users specifically?

Reply:
Good thought. Why not. Less hand-holding. We still need to enable EPEL repository for CentOS (as it is supported) and should offer (?) enabling it for all other distros.